core/embed/rust/src/ui/layout_eckhart/firmware/action_bar.rs

```diff
@@ -81,8 +79,13 @@ impl ActionBar {
             Mode::Double {
                 pager: Pager::single_page(),
             },
-            Some(left.with_expanded_touch_area(Self::BUTTON_EXPAND_TOUCH)),
-            right.with_expanded_touch_area(Self::BUTTON_EXPAND_TOUCH),
+            Some(
+                left.with_expanded_touch_area(Self::BUTTON_EXPAND_TOUCH)
+                    .with_content_offset(Self::BUTTON_CONTENT_OFFSET),
+            ),
+            right
+                .with_expanded_touch_area(Self::BUTTON_EXPAND_TOUCH)
+                .with_content_offset(Self::BUTTON_CONTENT_OFFSET * -1.0),
```

Comment:
There is a function to negate an offset:
```suggestion
                .with_content_offset(Self::BUTTON_CONTENT_OFFSET.neg()),
```

Reply:
https://github.com/trezor/trezor-firmware/pull/4708/commits/d0304625154e88dee361cadb01a8191dd0794b02